controllers/controller.js

```diff
@@ -0,0 +1,4 @@
+const { getCategories } = require("./categoriesControllers");
```

Comment:
You can just import/export directly from your individual controller/model files. No need to have a general controller file to bridge them 👍